tembo-cli/src/cmd/apply.rs

```diff
@@ -513,14 +530,14 @@ fn get_update_instance(instance_settings: &InstanceSettings) -> UpdateInstance {
         replicas: instance_settings.replicas,
         app_services: None,
         connection_pooler: None,
-        extensions: Some(Some(get_extensions(instance_settings.extensions.clone()))),
+        extensions: Some(Some(get_extensions(instance_settings.extensions.clone())?)),
```

Comment:
I think this is used on update, but it's using always the latest version of an extension found in trunk. It is not safe to assume the version should be set to latest, it should be set to the version that's already enabled in the case where the extension is already turned on.

Reply:
Consider this case:

- user does not specify version
- deploy to cloud
- new version of extension published to trunk
- deploy again to same instance

Reply:
In the above case, it must include the previous version, not the newly published version